src/Http.php

```diff
@@ -117,7 +119,7 @@ public function request( $method, $url, $headers = array(), $params = array() )
                 $this->curl->setOpt( $ch, CURLOPT_POSTFIELDS, $params );
 
                 if ($this->postDataFormat == 'json') {
-                    array_push( $headers, 'Accept: application/json', 'Content-Type: application/json' );
+                    array_push( $headers, 'Content-Type: application/json' );
```

Comment:
The accept header was sent only when there were either GET parameters or POST body set. This was a wrong place to set the header. It should be probably sent in all the requests, I'll fix that.